static/app/views/performance/http/useStatusPageList.tsx

```diff
@@ -0,0 +1,22 @@
+import {useCallback, useEffect, useState} from 'react';
+
+import {useIsMountedRef} from 'sentry/utils/useIsMountedRef';
+
+export function useStatusPageList() {
+  const isMountedRef = useIsMountedRef();
+  const [mod, setMod] = useState<any>({});
+
+  const loader = useCallback(async () => {
+    const loaded = await import('@sentry/status-page-list');
```

Comment:
seems excessive to load what an object with 50 things in it, assuming we can tree shake the rest of it. We might need this https://github.com/getsentry/status-page-list/pull/6

Performance page routes are already lazy loaded.

Reply:
what i'm saying is it might be fine to just import {domainToStatusPageUrls} from '@sentry/status-page-list'

Reply:
@scttcper I'm happy to follow whatever you think is best!

The docs in `status-page-list` suggested to load async, which seems sound since that package is going to grow and we probably can't tree-shake it, since we don't know which domains we need ahead of time, right?

Reply:
yeah was thinking mostly of shaking everything else

Reply:
Ah, roger! Will update the dependency if/when https://github.com/getsentry/status-page-list/pull/6 is merged, then